scripts/inject.js

```diff
@@ -1,45 +1,61 @@
 import { rollupScript } from './utils/build.js'
+import { parseArgs, write } from './script-utils.js'
 
 const contentScopePath = 'src/content-scope-features.js'
 const contentScopeName = 'contentScopeFeatures'
 
-async function init () {
-    if (process.argv.length !== 3) {
-        throw new Error('Specify the build type as an argument to this script.')
-    }
-    if (process.argv[2] === 'firefox') {
-        initOther('inject/mozilla.js', process.argv[2])
-    } else if (process.argv[2] === 'apple') {
-        initOther('inject/apple.js', process.argv[2])
-    } else if (process.argv[2] === 'android') {
-        initOther('inject/android.js', process.argv[2])
-    } else if (process.argv[2] === 'windows') {
-        initOther('inject/windows.js', process.argv[2])
-    } else if (process.argv[2] === 'integration') {
-        initOther('inject/integration.js', process.argv[2])
-    } else if (process.argv[2] === 'chrome-mv3') {
-        initOther('inject/chrome-mv3.js', 'chrome_mv3')
-    } else {
-        initChrome()
+/**
+ * @param {string} entry
+ * @param {string} platformName
+ */
+async function bundle (entry, platformName) {
+    if (platformName === 'firefox') {
+        return initOther(entry, platformName)
```

Comment:
Do we need to expose 'entry' to the build script? The risk of pointing to the wrong file is pretty catastrophic.

Reply:
let's put both entry + output into the script?  I'd prefer that anyway, easier to track changes over time too in git